src/lib.rs

```diff
@@ -7,11 +7,11 @@
 
 #[cfg(feature = "clipboard")]
 use clipboard::{ClipboardContext, ClipboardProvider};
-use egui::Key;
 use egui::{
     math::{pos2, vec2},
     CtxRef,
 };
+use egui::{paint::ClippedShape, Key};
```

Comment:
Please also move the import into the already existing "egui::" braket in line 10.